### PR TITLE
add error handling

### PR DESCRIPTION
**Pull request by toshi0607**
* add error handling
* gofmt 3 files

### CODE REVIEW

File: cmd/cloudshell_open/main.go [hunk 1]
```diff
@@ -311,15 +316,13 @@ func checkCloudShellTrusted() (bool, error) {
 	return false, fmt.Errorf("error determining if cloud shell is trusted: %+v. output=\n%s", err, string(b))
 }
 
-func waitForBilling(projectID string, prompt func(string)) error {
-	for {
```
Comment (ahmetb) [inline cmd/cloudshell_open/main.go]: looks like you've removed the for-loop here. why?

Comment (toshi0607) [reply cmd/cloudshell_open/main.go]: I thought this was unintentional, but is this intentional for waiting until billing will be enabled and is infinite loop OK if billing will not be enabled?

Comment (toshi0607) [reply cmd/cloudshell_open/main.go]: If this loop is intentional, I will revert loop removal and return result of prompt only when error happened, of course!

Comment (ahmetb) [reply cmd/cloudshell_open/main.go]: Yeah this is intentional. It would be good to preserve the logic.

Comment (toshi0607) [reply cmd/cloudshell_open/main.go]: Thank you! I'll fix!

**Comment by ahmetb** [comment]
Thanks, looks good.

I'm not entirely sure how the error returned error will be used in this case. I suspect we'll start seeing error messages like `EOF` or `interrupt` due to these when user hits Ctrl+C or something like that.

At least we're not ignoring it now.

**Pull request merged**